cockatrice/src/client/network/replay_timeline_widget.h

```diff
@@ -18,6 +18,8 @@ class ReplayTimelineWidget : public QWidget
 
 private:
     QTimer *replayTimer;
+    static const int REWIND_BUFFERING_TIMEOUT_MS = 180;
```

Comment:
in my testing with a long replay this seems a bit short still, cockatrice seems to take 50% single core if I tap back slower than 5 times per second

Reply:
should we just make this value configurable so we don't have to settle on a hardcoded value?

Reply:
we could make it proportional to the total length of the replay, or how far into the replay it is, or how estimate based on the total length of the replay (note that not all commands take the same amount of cycles to process)